package.json

```diff
@@ -197,6 +197,9 @@
 		"packages/*/package.json": [
 			"wp-scripts lint-pkg-json"
 		],
+		"*.scss": [
+			"stylelint"
```

Comment:
@ntwb, it would be nice to include `stylelint` and `eslint` in `wp-scripts` as new commands 😄